src/commands/deploy/nextjs/deploy.ts

```diff
@@ -253,10 +283,12 @@ async function deployCDN(
     };
 
     const paths = [
+        ...externalPaths,
         { origin: serverOrigin, pattern: "api/*" },
         { origin: serverOrigin, pattern: "_next/data/*" },
         { origin: imageOptimizationOrigin, pattern: "_next/image*" },
     ];
+
     const assetsFolder = path.join(process.cwd(), ".open-next", "assets");
     paths.push(...(await computeAssetsPaths(assetsFolder, s3Origin)));
```

Comment:
Ia there any chance that these edge functions paths might collide with the assets paths?